diff --git a/caja.spec b/caja.spec
index 18fed95..87e0bdf 100644
--- a/caja.spec
+++ b/caja.spec
@@ -69,7 +69,7 @@ It is also responsible for handling the icons on the MATE desktop.
 %package core-extensions
 Summary:  Mate-file-manager extensions library
 Requires: %{name}%{?_isa} = %{version}-%{release}
-%if (0%{?fedora} >= 28 && 0%{?fedora} <= 32)
+%if (0%{?fedora} >= 28 && 0%{?fedora} <= 32) || (0%{?rhel} >= 7)
 Provides: %{name}-extensions = %{version}-%{release}
 Provides: %{name}-extensions%{?_isa} = %{version}-%{release}
 Obsoletes: %{name}-extensions < 1.20.2-2
